pkg/broker/work_engine.go

```diff
@@ -93,17 +143,17 @@ func (engine *WorkEngine) AttachSubscriber(
 		return errors.New("invalid work topic")
 	}
 
-	msgBuffer, topicExists := engine.topics[topic]
-	if !topicExists {
-		msgBuffer = make(chan JobMsg, engine.bufsz)
-		engine.topics[topic] = msgBuffer
-	}
+	engine.subscribers[topic] = append(engine.subscribers[topic], subscriber)
 
-	subscriber.Subscribe(msgBuffer)
 	return nil
 }
 
-// GetActiveTopics - Get list of topics
-func (engine *WorkEngine) GetActiveTopics() map[WorkTopic]chan JobMsg {
-	return engine.topics
+// GetActiveJobs - Get list of active jobs
+func (engine *WorkEngine) GetActiveJobs() map[string]chan JobMsg {
+	return engine.jobs
```

Comment:
is this really active jobs? I would expect it to return an array of jobs.

Reply:
changed to GetActiveJobChannels